validphys2/src/validphys/config.py

```diff
@@ -1039,6 +1039,17 @@ def produce_t0set(
                 raise ConfigError("Setting use_t0 requires specifying a valid t0pdfset")
             return t0pdfset
         return None
+    
+    def parse_luxset(self, name):
+        """PDF set used to generate the photon with fiatlux."""
+        return self.parse_pdf(name)
+    
+    def parse_additional_errors(self, bool):
+        """PDF set used to generate the photon additional errors."""
+        if bool:
+            return self.parse_pdf("LUXqed17_plus_PDF4LHC15_nnlo_100")
```

Comment:
Instead of putting the name here maybe you could make separate actions similar to what is done for t0:
- `def parse_additional_errors(name)`
- `def parse_use_additional_errors(bool)`

Then implement a check that if `use_additional_errors==True` a pdfset must be given for `additional_errors` otherwise raise a `ConfigError`.

Reply:
You mean that I should pass the additional errors pdfset from the runcard? I'm skeptical about it since this pdfset must always be `LUXqed17_plus_PDF4LHC15_nnlo_100` so I would hardcode it (moreover since we use the replicas from 101 to 107 if the set we pass from runcard has e.g. 100 replicas it will cause problems)

Reply:
Hmm, `ErrorType` is not set correctly in that PDF so there's no way to generalise which members to take. I just don't like the fact that an external input is hardcoded and thus somewhat hidden from the user, though I'm inclined to accept your proposal.

Perhaps @Zaharid or @scarlehoff have an opinion on this.